tests/integration/components/feed-item-test.js

```diff
@@ -9,7 +10,43 @@ test('it renders', function(assert) {
   // Set any properties with this.set('myProperty', 'value');
   // Handle any actions with this.on('myAction', function(val) { ... });
 
-  this.render(hbs`{{feed-item}}`);
+  let article = {
+    author: {
+      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
+      username: 'Alon Bukai'
+    },
+    createdAt: moment('1995-12-25'),
+    favoritesCount: 9000,
+    body: 'This is feed item content',
+    slug: 'feed-item-title',
+    title: 'Feed Item Title'
+  };
 
-  assert.equal(this.$().text().trim(), '');
+  this.set('article', article);
+  this.render(
+    hbs`{{feed-item 
+            imageSrc=article.author.image 
+            author=article.author.username 
```

Comment:
If the `author` property is actually supposed to be their name, we should pass it in as such.

Alternatively, and what I think makes more sense, would be to just pass in an `author` and `article` object (or maybe just the `article` since the author is likely attached) rather than having this component configured with all these properties.